packages/mobile/src/exchange/WithdrawCeloScreen.tsx

```diff
@@ -25,14 +25,20 @@ import { HeaderTitleWithBalance, headerWithBackButton } from 'src/navigator/Head
 import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
 import useSelector from 'src/redux/useSelector'
+import { getSendFee } from 'src/send/saga'
 import { useDailyTransferLimitValidator } from 'src/send/utils'
 import DisconnectBanner from 'src/shared/DisconnectBanner'
+import { divideByWei } from 'src/utils/formatting'
+import Logger from 'src/utils/Logger'
 
 type Props = StackScreenProps<StackParamList, Screens.WithdrawCeloScreen>
 
+const RANDOM_ADDRESS = '0xDCE9762d6C1fe89FF4f3857832131Ca18eE15C66'
```

Comment:
We don't have an address here yet to do the estimate, and estimating requires an address, so I used a random one. Could that cause any issues?

Reply:
This is fine, we already do this [here as well](https://github.com/celo-org/celo-monorepo/blob/2588f4de9691b2944e8f18069d3cf44a77bb3ec1/packages/mobile/src/fees/saga.ts#L24).